.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41B Bug report"
+about: Submit a bug report to improve our library!
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+** Environment **
+<!-- Provide the below as necessary -->
+
+- OS: [Ubuntu 20.04]
+- Hardware (GPU, or instance type): [V100]
```

Comment:
H100

Reply:
Done in https://github.com/mosaicml/streaming/pull/48/commits/5ce40e3815fe964e4de8e44ad5f05dc0dc40dc98.